.github/workflows/test-sample-go.yml

```diff
@@ -0,0 +1,31 @@
+name: project-v4-sample
+
+# Trigger the workflow on pull requests and direct pushes to any branch
+on:
+  push:
+  pull_request:
+
+jobs:
+  test:
+    name: ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}

```

Comment:
We do not need a matrix.os here
We just do the test using ubuntu linux for this one.